homeassistant/components/braviatv/config_flow.py

```diff
@@ -22,30 +27,32 @@
 )
 
 
-def host_valid(host):
+def host_valid(host: str) -> bool:
     """Return True if hostname or IP address is valid."""
     try:
         if ipaddress.ip_address(host).version in [4, 6]:
             return True
     except ValueError:
-        disallowed = re.compile(r"[^a-zA-Z\d\-]")
-        return all(x and not disallowed.search(x) for x in host.split("."))
+        pass
```

Comment:
Instead of try/except->pass, please use contextlib.suppress

Reply:
Done